R/UnsupervisedLumberjack.R

```diff
@@ -0,0 +1,97 @@
+#' Unsupervised Lumberjack
```

Comment:
Delete this file and remove references in Namespace (from discussion with James this function doesn't work and shouldn't have been committed)